libs/deal-ticket/src/components/expiry-selector.tsx

```diff
@@ -11,7 +11,7 @@ export const ExpirySelector = ({ value, onSelect }: ExpirySelectorProps) => {
   const dateFormatted = formatForInput(date);
   const minDate = formatForInput(date);
   return (
-    <FormGroup label="Expiry time/date" labelFor="expiration">
```

Comment:
chore (blocking): Can we leave this in and make sure the id of the input matches up?